modules/installation-and-upgrade/pages/install-vm.adoc

```diff
@@ -120,56 +242,16 @@ SUSEConnect -p sle-module-server-applications/15.4/x86_64
 SUSEConnect -p sle-module-web-scripting/15.4/x86_64
 SUSEConnect -p sle-module-suse-manager-server/<productnumber>/x86_64
```

Comment:
This command is already done above in lines 232. So maybe we remove this last one as well. 
```suggestion
```

Reply:
I think someone has already removed the previous occurrence.  At least,I cannot find it.